SETUP.md

```diff
@@ -41,3 +41,4 @@ And then open chrome at: `http://localhost:8080`
    }
  ]
 ``` 
+4) Configure `ROOMS_SOURCE` to `ENVIRONMEN` to get value 
```

Comment:
```suggestion
4) Configure `ROOMS_SOURCE` to `ENVIRONMENT` to get value 
```

Reply:
Hey, @enieber thank you for your contribution. Just a small typo fix as a suggestion.